examples/gdb.sh

```diff
@@ -0,0 +1 @@
+pkexec /usr/bin/gdb "$@"
```

Comment:
We should have a header comment explaining what this file is for

Reply:
Not sure I entirely understand the purpose of this file. I think it allows administrator privileges for gdb, which is why the pkexec command is used, but I don't think I can succinctly summarize it in a comment

Reply:
Here is a suggestion:

```suggestion
# This is a wrapper script for GDB that allows it to run as root. This is necessary if you are calling
# gdb from an IDE like VS Code that by itself does not run as root.
# NOTE: In some cases we have found that you must use `sudo` instead of `pkexec` here.
pkexec /usr/bin/gdb "$@"
```

We will need to test this on a few different setups to understand whether to use `sudo` or `pkexec`. My memory is that when we tried to use this on Noah's machine it only worked with `sudo`, but on yours you needed `pkexec`. We will want to set this to whichever is needed on CloudLab since it is our "standard" platform.